### PR TITLE
create modification form for E24, use new form formatting

### PR DESCRIPTION
**Pull request by mradamcox**
### Description of Change
This PR creates a single new form for the E24 resource type, Heritage Feature. This is a brand new form (not copy/pasted from existing forms in the EAMENA code base), so I used a format system that I developed for the AFRH Arches implementation, a markup system that makes more use of bootstrap classes instead of style added at the DOM-element. A new stylesheet was added as well, which, in the future, could be merged into the existing `package.css`.

Note that you'll be able to view this form at http://localhost:8000/resources/HERITAGE_FEATURE.E24/modification/.

Also, this PR must be merged into a branch into which PR #6 has already been merged. Otherwise, you'll get an error when trying to view the form. If necessary, I can add a commit here to make it work without #6.

### CODE REVIEW

**Comment by mradamcox** [comment]
Unfortunately, MODIFACATION.E11 is actually the name of that particular node in the resource graphs, so that spelling error must be reflected in the html/js/py code for this form as well. @azerbini I'll leave you to confirm this and make the necessary changes.

**Comment by azerbini** [comment]
You are right, but I will take changing the table values over accepting a typo. I am replacing MODIFACATION.E11 with MODIFICATION.E11 across my ontology tables and in the data.entity_types table. 

```
update data.entity_types SET entitytypeid = 'MODIFICATION.E11' where entitytypeid = 'MODIFACATION.E11';
update ontology.mappings SET entitytypeidto = 'MODIFICATION.E11' where entitytypeidto = 'MODIFACATION.E11';
update ontology.rules SET entitytyperange = 'MODIFICATION.E11' where entitytyperange = 'MODIFACATION.E11';
update ontology.rules SET entitytypedomain = 'MODIFICATION.E11' where entitytypedomain = 'MODIFACATION.E11';
```

**Comment by azerbini** [comment]
Nevermind, there are a bunch of constraints that won't let you do that. I think the only thing that is viable  then is to unload the E24 graph and reload it again then.

File: eamena/eamena/templates/views/forms/modification.htm [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+{% block form_content %}
+<div class="form-full-content">
+    <div class="row">
+        <div class="col-xs-12 margin-bottom-10">
+            <div class="form-section-header"><span>{% trans "Modifications" %}</span></div>
+        </div>
+    </div>
+    <div id="modification-section">
```
Comment (azerbini) [inline eamena/eamena/templates/views/forms/modification.htm:13]: @mradamcox This is not one section, but rather should be separated out into three, as the E24 Interface doc explains:

> 1. Type of Modification (Modification_Type.E55) & Date of Modification (Modification_Date.E49)
> 2. Modification Remarks (Modification_Description.E62) 
> 3. Construction Material (Construction_Material.E57) & Construction Technique (Construction_Technique_Type.E55) & Note (Construction_Note.E62) 


Each of the three above should be given a separate section

Comment (mradamcox) [reply eamena/eamena/templates/views/forms/modification.htm:13]: @azerbini, can you point me to that document? I don't think I have received/seen it.

**Pull request merged**